app/src/main/java/com/besome/sketch/editor/view/ViewEvents.java

```diff
@@ -142,4 +172,11 @@ public int getItemCount() {
             return events.size();
         }
     }
+    
+    public void deleteEvent(EventBean event) {
+    	jC.a(sc_id).d(projectFileBean.getJavaName(), event.targetId, event.eventName);
+        eC a2 = jC.a(sc_id);
+        String javaName = projectFileBean.getJavaName();
+        a2.k(javaName, event.targetId + "_" + event.eventName);
+    }
```

Comment:
> this seems like a copy of `a.a.a.rs#deleteEvent(EventBean)`. don't do that, that causes code duplication which should be prevented.
> 
> it would've made more sense to create a new method in `EventBean` or `ProjectFileBean` to delete an event, with sc_id and either a `ProjectFileBean` or `EventBean` as parameters. then remove the old method in `a.a.a.rs` and make it use the new one you just created though.

EventBean will be better but as static method or non-static?